internal/bundler/bundler_test.go

```diff
@@ -9,11 +10,14 @@ import (
 	"github.com/evanw/esbuild/internal/parser"
 	"github.com/evanw/esbuild/internal/printer"
 	"github.com/evanw/esbuild/internal/resolver"
+	"github.com/kylelemons/godebug/diff"
 )
 
 func assertEqual(t *testing.T, a interface{}, b interface{}) {
 	if a != b {
-		t.Fatalf("%s != %s", a, b)
+		stringA := fmt.Sprintf("%v", a)
+		stringB := fmt.Sprintf("%v", b)
+		t.Fatalf(diff.Diff(stringA, stringB))
```

Comment:
i've added this change because it's hard to diff any whitespace differences in tests when it happens